controllers/selfnoderemediation_controller.go

```diff
@@ -75,9 +75,10 @@ var (
 type processingChangeReason string
 
 const (
-	remediationStarted         processingChangeReason = "RemediationStarted"
-	remediationTerminatedByNHC processingChangeReason = "RemediationStoppedByNHC"
-	remediationFinished        processingChangeReason = "RemediationFinished"
+	remediationStarted              processingChangeReason = "RemediationStarted"
+	remediationTerminatedByNHC      processingChangeReason = "RemediationStoppedByNHC"
```

Comment:
NIT: I would follow the pattern of the const name match its value by updating the const name to be `remediationStoppedByNHC `

Reply:
and NHC is signalling a timeout actually, not stopping or terminating anything 😉

Reply:
Too many verbs for one action :D